hackathon_site/hardware/views.py

```diff
@@ -17,11 +19,18 @@
 
 logger = logging.getLogger(__name__)
 
+from hardware.api_filters import HardwareFilter
+from hardware.models import Hardware, Category, Order
```

Comment:
This is a duplicate import now, since the same thing was done when merging in develop (line 10). After doing a merge, make sure to open up the files and check for unused imports, which could mean something went wrong when resolving the conflicts

Reply:
got it!